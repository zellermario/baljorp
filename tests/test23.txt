create SJ mezo 3

create TorekenyAso ta field mezo
create Eszkimo e mezo

digobject e
useobject ta e