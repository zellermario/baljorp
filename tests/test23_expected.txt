Fields:
SJ mezo (1) - object: ?
	neighbours: []

Buildings:

Actors:
> Eszkimo e (heat: 5) on mezo
  Inventory: [ta]